src/lib/containers/home.tsx

```diff
@@ -85,7 +85,10 @@ export class Home extends React.Component<Props, State> {
   }
 
   componentDidUpdate() {
-    this.listView.scrollTo({ y: this.currentScrollOffset + 1, animated: false })
+    if (this.currentScrollOffset < 0) {
```

Comment:
It is triggered both before and after the request (the first `componentDidUpdate` console log there happens immediately after you release the drag gesture)

![screen shot 2017-05-17 at 3 39 05 pm](https://cloud.githubusercontent.com/assets/2712962/26156583/15a68894-3b17-11e7-9ec0-4faff3ab2389.png)